pkg/resource/logging-config/alloy/logging-config.alloy.yaml.template

```diff
@@ -58,3 +61,60 @@ alloy:
           path: /run/log/journal
       - name: alloy-tmp
         emptyDir: {}
+podLogs:
+{{- if .IsWorkloadCluster }}
+- name: namespace-default
+  namespace: giantswarm
+  spec:
+    selector: {}
+    namespaceSelector:
+      matchExpressions:
+      - key: kubernetes.io/metadata.name
+        operator: In
+        values:
+        - kube-system
+        - giantswarm
+- name: pod-selector
+  namespace: giantswarm
+  spec:
+    selector:
+      matchLabels:
+        giantswarm.io/logging: enabled
+    namespaceSelector:
+      matchExpressions:
+      - key: giantswarm.io/logging
+        operator: NotIn
+        values:
+        - enabled
```

Comment:
Could WE not simple use giantswarm.io/logging: true liké we do for clusters?

Reply:
Did you test this with logging disabled at the cluster level ?

Reply:
Good point, I haven't tested it with logging disabled at cluster level.

Sure we can use true/false instead of enabled/disabled

Reply:
I changed the values to true/false.

Also tested with logging disabled at the cluster level, Alloy is uninstalled so it works :heavy_check_mark: